scripts/data_prep/convert_delta_to_json.py

```diff
@@ -676,6 +676,9 @@ def fetch_DT(args: Namespace) -> None:
         log.info(f'Elapsed time {time.time() - tik}')
 
     except Exception as e:
-        if mosaicml_logger is not None:
+        if mosaicml_logger is not None and os.environ.get(
```

Comment:
can we add a helper function somewhere for the environ check? otherwise its a pretty easy condition to make a mistake on.

Reply:
This is reminding me that we need some end to end tests for custom exceptions, will work with Vincent to add those